q2_sample_classifier/plugin_setup.py

```diff
@@ -226,15 +228,20 @@
              ('feature_importance', FeatureData[Importance]),
              ('predictions', SampleData[ClassifierPredictions])
              ] + pipeline_outputs + [
-                ('probabilities', SampleData[Probabilities]),
-                ('heatmap', Visualization)],
+        ('probabilities', SampleData[Probabilities]),
+        ('heatmap', Visualization),
+        ('training_targets', SampleData[TrueTargets]),
+        ('test_targets', SampleData[TrueTargets])],
     input_descriptions={'table': input_descriptions['table']},
     parameter_descriptions=classifier_pipeline_parameter_descriptions,
     output_descriptions={
         **pipeline_output_descriptions,
         'probabilities': input_descriptions['probabilities'],
         'heatmap': 'A heatmap of the top 50 most important features from the '
-                   'table.'},
+                   'table.',
+        'training_targets': 'Table containing true target values of'
```

Comment:
I would not refer to this as a table (to avoid confusion with "feature table")... instead perhaps a list of true target values?

Reply:
sounds good!